packages/ember-views/tests/views/view/init_test.js

```diff
@@ -30,7 +30,7 @@ test("registers view in the global views hash using layerId for event targeted",
 
 QUnit.module("EmberView.createWithMixins");
 
-test("should warn if a non-array is used for classNames", function() {
+test("should warn if a computed property is used for classNames", function() {
```

Comment:
See my comment above. I tested strings, integers, regexes, Booleans, etc. as values for `classNames`, and none of them were caught by the `assert`. Only using a computed property is caught by the `assert`. So my goal with this PR was simply to clarify what this test is *actually* testing. I can do a separate PR to test that `classNames` is actually an array.